test/uk/gov/hmrc/testlogin/services/LoginServiceSpec.scala

```diff
@@ -38,15 +38,9 @@ class LoginServiceSpec extends AsyncHmrcSpec with BeforeAndAfterAll {
   trait Setup {
     implicit val hc: HeaderCarrier   = HeaderCarrier()
     val apiPlatformTestUserConnector = mock[ApiPlatformTestUserConnector]
-    val underTest                    = new LoginService(apiPlatformTestUserConnector)
-  }
-
-  override def beforeAll(): Unit = {
-    setCurrentMillisFixed(10000)
-  }
-
-  override def afterAll(): Unit = {
-    setCurrentMillisSystem()
+    val now                          = Instant.now()
```

Comment:
can grab this from FixedClock inside of the `api-platform-test-common-domain`

Reply:
I wasn't sure whether to bring in the common domain for this.  Have done so now...